src/batch/src/executor/iceberg_scan.rs

```diff
@@ -64,6 +68,7 @@ impl IcebergScanExecutor {
         snapshot_id: Option<i64>,
         table_meta: TableMetadata,
         file_scan_tasks: Vec<FileScanTask>,
```

Comment:
how about rename it to "data_file_scan_tasks".

So we can dishtingush the "data file"  and "eq delete file"